endpoints/citrus-http/src/main/java/org/citrusframework/http/servlet/CachingHttpServletRequestWrapper.java

```diff
@@ -129,15 +128,8 @@ private void fillParams(final Map<String, String[]> params, final String querySt
                     paramValue = "";
                 }
 
-                try {
-                    params.put(URLDecoder.decode(paramName, charset.name()),
-                            new String[] { URLDecoder.decode(paramValue, charset.name()) });
-                } catch (final UnsupportedEncodingException e) {
-                    throw new CitrusRuntimeException(String.format(
-                            "Failed to decode query param value '%s=%s'",
-                            paramName,
-                            paramValue), e);
-                }
+                params.put(URLDecoder.decode(paramName, charset),
```

Comment:
wondering if this is safe? `CitrusRuntimeException` could possibly be catched somewhere, whereas the `UnsupportedEncodingException` wouldn't be.

Reply:
The exception is never thrown from the decode if you pass the charset directly. So this is OK.